clusters/hive/pools/ocp-release-4.9.0-rc.1-x86-64-for-4.9.0-0-to-4.10.0-0_clusterimageset.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterImageSet
+metadata:
+  annotations:
+    version_lower: 4.9.0-0
```

Comment:
Does the range include `4.9.0-rc.1`? Currently, 4.9 has no GA versions.

Reply:
The `clusterimageset-updater` uses the `prerelease` release type, which includes all releases in the `4-stable` imagestream, which includes the `fc` and `rc` releases: https://amd64.ocp.releases.ci.openshift.org/#4-stable

Reply:
Is this correct `4.9.0-0` < `4.9.0-rc.1` < `4.10.0-0` according to the tool?
To me, `4.9.0-rc.1` < `4.9.0-0` because the former is prerelease and the latter is released.

Reply:
The `4.X.0-0` releases are non-existent releases that are technically before any releases in the imagestream. The first non-fc and non-rc release of OCP versions are just `4.X.0`, without anything after the micro version. So `4.X.0-` versions are considered to be before the `4.X.0` versions. The ordering is numerical and alphabetical (so first `0`, then the `fc.X`, then `rc.X`). This is also why the upper limit is `4.10.0-0` instead of `4.10.0`, as `4.10.0` would result in the imagestream being updated to the `4.10` `fc` and `rc` versions, which are before `4.10.0`.